kube/services/revproxy/gen3.nginx.conf/viewer-service.conf

```diff
@@ -0,0 +1,10 @@
+location /viewer/ {
```

Comment:
can we call it "dicom-viewer" and "dicom-viewer-service" everywhere instead of "viewer" and "viewer-service"?